tests/integration.rs

```diff
@@ -2506,36 +2507,43 @@ mod tests {
         }
 
         #[test]
+        #[cfg(target_arch = "x86_64")]
         fn test_vhost_user_net_self_spawning() {
```

Comment:
Why do these tests not work? is vhost-user not supported?

Reply:
Emmm some vhost_user test cases failed on both my AArch64 dev machine and the CI machine for following two reasons:

1. Any vhost_user_net case with self-spawning will fail everytime. The VM just cannot be started...

2. the `test_vhost_user_blk_default` and `test_vhost_user_blk_direct` case will randomly fail (1 or 2 fail out of 10). I have literally no idea why this happened. If I run these two cases with exact same configuration and argument on my host manually, it works everytime. However in container the ssh command will randomly fail, leading to the assertion fail. I checked the log and found that the kernel is started....So I am really confused and still debugging this.

As this PR is blocking the PCI related integration test. We will propose another PR after the above bugs are resolved.

Reply:
@rbradford Some failed error log can be found here: https://cloud-hypervisor-jenkins.westus.cloudapp.azure.com/blue/organizations/jenkins/cloud-hypervisor/detail/PR-1463/17/pipeline/137

As this failure is really random so I really don't feel comfortable to open these two cases...

Reply:
I'm fine with tests being disabled for now. I just wondered if this was a particular vhost-user issue but sounds not.